index.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly consider avoiding disabling linters because it enforces coding standards and guidelines, ensuring that code follows a consistent style throughout a project or among a team of developers. Consistent code improves readability and makes it easier for developers to understand and maintain the codebase.